_conferences/eurosys2022/results.md

```diff
@@ -101,36 +112,42 @@ artifacts:
     paper_url: "10.1145/3492321.3519557"
     artifact_url: "10.5281/zenodo.6347456"
     repository_url: "https://github.com/SJTU-IPADS/fgnn-artifacts/tree/0abc7182"
+    summary: "gnnlab"
 
   - title: "VMSH: Hypervisor-agnostic Guest Overlays for VMs"
     badges: "available,functional,reproduced"
     award: "Distinguished Artifact Honorable Mention"
     paper_url: "10.1145/3492321.3519589"
     artifact_url: "10.5281/zenodo.6337102"
     repository_url: "https://github.com/Mic92/vmsh"
+    summary: "vmsh"
 
   - title: "Kite: Lightweight Critical Service Domains full strip note"
     badges: "available,functional,reproduced"
     paper_url: "10.1145/3492321.3519586"
     artifact_url: "10.5281/zenodo.6348174"
     repository_url: ""

```

Comment:
Should this be filled in, since the badges include available?

Reply:
repositories are not required, the artifact is available via DOI/Zenodo